winui/Shadow/Getting-Started.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Getting Started with WinUI Shadow control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Shadow(SfShadow) control, its elements, and more.
```

Comment:
description: Learn all about getting started with the Syncfusion WinUI Shadow(SfShadow) control, its elements, and more here.

Reply:
Changed the description in Getting-started as mentioned above.